tests/main/preseed-core20/create-key.exp

```diff
@@ -0,0 +1,17 @@
+spawn snap create-key
+
+expect "Passphrase: "
+sleep .5
+send "pass\n"
+
+expect "Confirm passphrase: "
+sleep .5
+send "pass\n"
+
+set status [wait]
+if {[lindex $status 3] != 0} {
+    exit 1
+}
+
+set timeout 60
```

Comment:
I think this might not be needed, I use in prepare as a slightly convoluted way of having gpg initialize its directory, otherwise I had issues with import. Maybe there is a better way but I need to experiment with this a bit.